lib/command.rb

```diff
@@ -16,34 +17,36 @@ def build
   private
 
   def base_branch
-    config.fetch("base_branch", "origin/master")
+    config["base_branch"] || github_data.base_ref || "origin/master"
   end
 
   def base_command
     "rubocop --parallel -f json"
   end
 
   def check_scope
-    return "git diff #{base_branch} --name-only --diff-filter=AM | xargs" if config["check_scope"] == "modified"
+    return unless config["check_scope"] == "modified"
+
+    "git diff #{base_branch}... --name-only --diff-filter=AM | xargs"
```

Comment:
The triple dot is the syntax that does the trick here. It also plays nicely with specifying a different `base_branch`